src/atoms/exp_cone/exp.jl

```diff
@@ -12,48 +12,48 @@ export sign, curvature, monotonicity, evaluate
 ### Exponential
 
 struct ExpAtom <: AbstractExpr
-  head::Symbol
-  id_hash::UInt64
-  children::Tuple{AbstractExpr}
-  size::Tuple{Int, Int}
-
-  function ExpAtom(x::AbstractExpr)
-    if sign(x)==ComplexSign()
-      error("The argument should be real but it's instead complex")
-    else
-      children = (x,)
-      return new(:exp, hash(children), children, x.size)
+    head::Symbol
+    id_hash::UInt64
+    children::Tuple{AbstractExpr}
+    size::Tuple{Int, Int}
+
+    function ExpAtom(x::AbstractExpr)
+        if sign(x)==ComplexSign()
+            error("The argument should be real but it's instead complex")
+        else
+            children = (x,)
+            return new(:exp, hash(children), children, x.size)
+        end
     end
-  end
 end
 
 function sign(x::ExpAtom)
-  return Positive()
+    return Positive()
 end
 
 function monotonicity(x::ExpAtom)
-  return (Nondecreasing(),)
+    return (Nondecreasing(),)
 end
 
 function curvature(x::ExpAtom)
-  return ConvexVexity()
+    return ConvexVexity()
 end
 
 function evaluate(x::ExpAtom)
-  return exp.(evaluate(x.children[1]))
+    return exp.(evaluate(x.children[1]))
 end
 
 exp(x::AbstractExpr) = ExpAtom(x)
 
 function conic_form!(e::ExpAtom, unique_conic_forms::UniqueConicForms=UniqueConicForms())
-  if !has_conic_form(unique_conic_forms, e)
-    # exp(x) \leq z  <=>  (x,ones(),z) \in ExpCone
-    x = e.children[1]
-    y = Constant(ones(size(x)))
-    z = Variable(size(x))
-    objective = conic_form!(z, unique_conic_forms)
-    conic_form!(ExpConstraint(x, y, z), unique_conic_forms)
-    cache_conic_form!(unique_conic_forms, e, objective)
-  end
-  return get_conic_form(unique_conic_forms, e)
+    if !has_conic_form(unique_conic_forms, e)
+        # exp(x) \leq z  <=>    (x,ones(),z) \in ExpCone
```

Comment:
Comment changed

Reply:
Sigh. My methodology was not infallible; I used
```vim
:set ts=2 sts=2 noet | retab! | set ts=4 sts=4 et | retab!
```
in Vim for each file then eyeballed the results. That expands every occurrence of 2 spaces to 4 spaces...